tests/columns/test_date.py

```diff
@@ -23,3 +23,25 @@ def test_do_not_use_timezone(self):
             with patch.dict(os.environ, {'TZ': 'US/Hawaii'}):
                 inserted = self.client.execute(query)
                 self.assertEqual(inserted, data)
+
+    def test_insert_datetime_to_date(self):
+        with self.create_table('a Date'):
+            testTime = date(2019, 2, 20)
```

Comment:
It seems that datetime instance should be here.

Reply:
Sure, my mistake.